Neos.Flow/Documentation/TheDefinitiveGuide/PartV/ValidatorReference.rst

```diff
@@ -487,6 +487,7 @@ Arguments
 *********
 
 * ``regularExpression`` (string): The regular expression to use for validation, used as given
+* ``validationErrorMessage`` (string): The error message to show for validation, if the regular expression validation fails
```

Comment:
This file is auto generated from the code and should be updted automatically, so no need to manually change it it, that might just lead to errors with the generator. :)